pleskdistup/common/src/util.py

```diff
@@ -10,13 +10,14 @@
 def logged_check_call(cmd: typing.Union[typing.Sequence[str], str], **kwargs) -> str:
     log.info(f"Running: {cmd!r}. Output:")
 
-    # I beleive we should be able pass argument to the subprocess function
+    # I believe we should be able pass argument to the subprocess function
     # from the caller. So we have to inject stdout/stderr/universal_newlines
     kwargs["stdout"] = subprocess.PIPE
-    kwargs["stderr"] = subprocess.STDOUT
+    kwargs["stderr"] = subprocess.PIPE
     kwargs["universal_newlines"] = True
 
     stdout = []
+    stderr = []
     process = subprocess.Popen(cmd, **kwargs)
     while None is process.poll():
         if not process.stdout:
```

Comment:
Why is this logic asymmetric? Will this work correctly if the process only outputs to stderr (and a lot)?

Reply:
Changed the logic to line by line consumption, please recheck